parser/prism/Translator.cc

```diff
@@ -308,6 +308,24 @@ std::unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             return make_unique<parser::Restarg>(parser.translateLocation(loc), gs.enterNameUTF8(name),
                                                 parser.translateLocation(nameLoc));
         }
+        case PM_RETURN_NODE: {
+            auto returnNode = reinterpret_cast<pm_return_node *>(node);
+            pm_location_t *loc = &returnNode->base.location;
+
+            absl::Span<pm_node_t *> prismReturnValues;
+            if (auto returnValues = returnNode->arguments; returnValues != nullptr) {
+                prismReturnValues = absl::MakeSpan(returnValues->arguments.nodes,returnValues->arguments.size);
+            }
```

Comment:
This is a `PM_ARGUMENTS_NODE`, which is already handled in `PM_CALL_NODE`. TODO: deduplicate